.vib/openldap/goss/openldap.yaml

```diff
@@ -0,0 +1,41 @@
+file:
+  /opt/bitnami/openldap/etc/slapd.d:
+    exists: true
+    filetype: symlink
+    linked-to: /bitnami/openldap/slapd.d
+  /opt/bitnami/openldap/var/data:
+    exists: true
+    filetype: symlink
+    linked-to: /bitnami/openldap/data
+  /opt/bitnami/openldap/sbin/slapadd:
+    exists: true
+    filetype: symlink
+    linked-to: /opt/bitnami/openldap/sbin/slapd
+  /opt/bitnami/openldap/etc/slapd.conf.default:
+    exists: false
+  {{ range $package := .Vars.packages }}
+  {{ $package }}:
+    exists: true
+    filetype: symlink
```

Comment:
I am not sure about this, but I think we might skip checking the `filetype` as there is nothing on our side to ensure the libraries are generated as symlinks instead of regular files.

Reply:
You are right, it's removed now.